specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/apimusers.json

```diff
@@ -692,6 +692,28 @@
         "password": {
           "type": "string",
           "description": "User Password. If no value is provided, a default password is generated."
+        },
+        "confirmation": {
```

Comment:
This is optional yes?  So it doesn't introduce a breaking change.

Reply:
yes, optional.